tests/unit/nupic/algorithms/knn_classifier_test.py

```diff
@@ -29,6 +29,34 @@
 
 class KNNClassifierTest(unittest.TestCase):
 
+ 
+  
+  
+  
```

Comment:
@chetan51  told me to move this line down to line 36, so I did it! :-P jeez... ok, I'll put it back now... :-)